packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -119,14 +116,8 @@ export class DotnetPluginImpl implements PluginImpl {
     if (!ctx.projectSettings) {
       throw new NoProjectSettingError();
     }
-
-    const selectedCapabilities = (ctx.projectSettings?.solutionSettings as SolutionSettings)
```

Comment:
We can still create an instance of CodeTemplateInfo and pass it to scaffoldFromZipPackage(), instead of hardcode the logic in scaffoldFromZipPackage(). Because according to the method name scaffoldFromZipPackage(), it is a common method.

Reply:
Agreed

Reply:
I mean the CodeTemplateInfo class in the packages/fx-core/src/plugins/resource/bot/v2/interface/codeTemplateInfo.ts file, @hund030 if CodeTemplateInfo is a common interface for scaffolding, maybe we can refactor it later to avoid each plugin create a similar class for scaffold source code

Reply:
@eriolchan  I got you mean. 
Because the codeTemplateInfo is in bot plugin, so what I do is let the existing templateInfo in tab plugin back. 
I agreed with you about move this interface into the common pack.  I have not done this because this pr is inlcuding too much things. 
I can create a new PR to do this refactor,  also I think we should have a sync with Zhijie first.